app/client/src/app/pages/Home.js

```diff
@@ -321,7 +321,7 @@ class Home extends Component<Props> {
           </RightSection>
         </Main>
         <Footer>
-          <Copyright>© 2018 Saad Quadri</Copyright>
+            <Copyright>© 2018 <a href="https://github.com/saadq">Saad Quadri</a></Copyright>
```

Comment:
Seems like you accidentally added extra indentation here. If you run `npm run format`, it should automatically fix this as well as other formatting inconsistencies for all the other files.